backend/src/main/java/gov/cdc/usds/simplereport/service/OrganizationService.java

```diff
@@ -493,4 +497,19 @@ public List<UUID> getOrgAdminUserIds(UUID orgId) {
         .filter(Objects::nonNull)
         .collect(Collectors.toList());
   }
+
+  /**
+   * Method HARD DELETES an Okta group without touching any of the application organization data.
```

Comment:
Do we want to not touch the application org data though? Even if it is to clean up e2e orgs, wouldn't we want to clean the org up on the app side of things as well? 🤔

Reply:
Yep we do! I elected not to couple them since I was extending the code that we added in spec 2 that already soft deletes the application organizations. In the change to the actual Cypress tests, was planning on doing something like

```
after ( () => {
    cleanUpPreviousRunSetupData(currentSpecRunVersionName);
    cleanUpRunOktaOrgs(currentSpecRunVersionName);
})
```
so that the tests delete both at the same time on teardown. I elected to leave them loosely coupled here just to make things more extendable down the road since the test refactors are still ongoing. 

Happy to go back and refactor this together so that there's just the one e2e deletion method, but don't want to preemptively write that in in case we discover / something changes for the rest of the specs.

Reply:
Got it! Sounds good -- like the idea of revisiting if needed :)